docs/user-guide/building-assistants.rst

```diff
@@ -1016,11 +1147,24 @@ in the conversation below (extracted from a conversation on `Rasa X <https://ras
      - utter_direct_to_step2
 
 Here we can see the user has completed a few chitchat tasks first, and then ultimately
-asks how they can get started with Rasa X. The EmbeddingPolicy correctly predicts that
+asks how they can get started with Rasa X. The TEDPolicy correctly predicts that
 Rasa X should be explained to the user, and then also takes them down the getting started
 path, without asking all the qualifying questions first.
 
 Since the ML policy generalized well in this situation, it makes sense to add this story
 to your training data to continuously improve your bot and help the ML generalize even
 better in future. `Rasa X <https://rasa.com/docs/rasa-x/>`_ is a tool that can help
 you improve your bot and make it more contextual.
+
+.. note::
+    Here's a minimal checklist of files we modified to handle more complex contextual conversations:
+
+      - ``domain.yml``: 
+          - set featurization of slots used by stories to guide the flow of a conversation
+      - ``data/stories.md``: 
+          - write stories using featurized slots to guide the flow of a conversation
+          - add user conversations successfully handled by an ML policy as stories
+      - ``config.yml`` (in the ``policies`` section): 
+          - set ``max_history`` high enough to account for contextual patterns
+          - replace the MemoizationPolicy with the AugmentedMemoizationPolicy
+          - add the TEDPolicy
```

Comment:
not 100% sure it makes sense to have a checklist here as some of this section is a bit more arbitrary... Or what are your thougths on that?

Reply:
I agree - It's more like a set of recommendations in the section anyway. I'd rather take it out